app/assets/javascripts/actions/media_wiki_revisions_actions.js

```diff
@@ -0,0 +1,24 @@
+import { fetchClassFromRevisions } from '../utils/media_wiki_classes';
+import { getWikiMap } from '../utils/revision_utils';
+import { fetchReferencesAdded } from '../utils/media_wiki_references_utils';
+
+export const fetchRevisionsAndReferences = async (revisions, dispatch) => {
+  const wikiMap = getWikiMap(revisions);
+  fetchClassFromRevisions(wikiMap).then((assessments) => {
+    // eslint-disable-next-line no-console
+    console.log('Successfully fetched all page assessment information');
+    return dispatch({
+      type: 'RECEIVE_ASSESSMENTS',
```

Comment:
These types should be imported as constants, to follow the convention of the rest of the redux actions.